Lab 0/README.md

```diff
@@ -70,6 +70,8 @@ To view a local version of the Kubernetes dashboard and to deploy apps into your
 
    The plug-in is displayed in the results as `container-registry`.
 
+
+# Install Docker
```

Comment:
I think it should have a empty line after it otherwise technically the next line is still part of it. Github doesn't seem to think so, so there's that.